### ISSUE TITLE
USB and GPU info

### ISSUE THREAD

**Issue by Blisto91** [soerenmeier/linux-info#1]
Hi there! Cool little crate.

You might already have this on the roadmap but I was wondering if GPU and USB devices info is planned? 🙂

**Comment by soerenmeier** [comment]
Hey, thanks for the interest.
Yeah USB is on the roadmap, but something simple as listing all devices like `lsusb` and if possible they're corresponding mount location.

For USB and GPU what features/information would you need?

**Comment by Blisto91** [comment]
I don't necessarily need it, but i was talking with some people about making some driver management in Linux and stumbled upon this if one wanted a rust solution.
I'm a Linux newb still so was researching how to get GPU and usb information directly from the system.

All the basic information really needed would be the cards listed in `/sys/class/drm` (on arch at least) where one can pull memory, PCIe information, vendor and manufacturer I'd.
Which then could be looked up in the PCI.id database with a tool like [pci-id-parser](https://github.com/ilyazzz/pci-id-parser)
But I'm rambling about my findings now and you probably already know this 😁 
Tldr just the basic GPU information like name, kernel driver and such.

The usb stuff sounds fine. My thought there was if a person had a external GPU. But don't how it is treated by Linux.
